src/main/host/thread_ptrace.c

```diff
@@ -457,8 +673,24 @@ static void threadptrace_flushPtrs(Thread* base) {
     _threadptrace_flushPtrs(thread);
 }
 
+static const char* _regs_to_str(const struct user_regs_struct* regs) {
```

Comment:
I wonder if there's some C preprocessor magic that would compress this function. I'm not sure that it would be better, though...